lib/topify.js

```diff
@@ -44,12 +44,35 @@ _initPlatformCode();
 function topify(domWindow) {
   if (unsupportedPlatform) return console.error(`Unable to topify minvid window on unsupported window toolkit ${platform}.`); // eslint-disable-line no-console
 
-  const winPtrStr = getNativeHandlePtrStr(domWindow);
+  let winPtrStr = getNativeHandlePtrStr(domWindow);
   if (!winPtrStr) return console.error('Unable to get native pointer for window.'); // eslint-disable-line no-console
 
+  // test code - to test issue #619
+  if (platform === 'cocoa') {
+    // watch for X seconds to see if if winPtrStr changes. it only changes once, from what i see (sometimes), so after first change stop checking
```

Comment:
Rather than have the pointer check function auto-invoked at this point in the file, I'd call it from the bottom of _macTopify. Otherwise makes sense, not sure why it's not working